app/[key]/page.js

```diff
@@ -0,0 +1,74 @@
+// app/[id]/page.js
+"use client";
+import Link from "next/link";
+import { useParams } from "next/navigation";
+import { ContactsAPI } from "../data/contactAPI";
+import styles from '../page.module.css'
+import { useState, useEffect } from "react";
+
+
+ 
+export default function ContactCard() {
```

Comment:
its mandatory to call the folder like this or there's something more meaningful?